contracts/Pausable.sol

```diff
@@ -1,30 +1,31 @@
 pragma solidity >=0.4.21 <0.6.0;
 
-contract Pausable {
-  address public owner;
-  bool public paused;
+import "./Ownable.sol";
 
-  constructor() public {
-    owner = msg.sender;
-  }
+contract Pausable is Ownable {
+  event LogPaused(address account);
+  event LogUnpaused(address account);
```

Comment:
Which question, do you think a client user would ask itself about these events?

Reply:
Not really, they more concern about can they do `split` or `withdraw`, am I right?

Reply:
I was thinking that they would filter per their own address. I am driving at `indexed account` .

Reply:
right, got it, thanks. If they need to filter the log, we need to index that.

Reply:
Fixed by 8b2cde4ddbf946ae251592599ee7eb4b1161f7b5